mtools/mloginfo/sections/query_section.py

```diff
@@ -115,12 +122,12 @@ def run(self):
             stats['max'] = max(group_events) if group_events else 0
             stats['mean'] = 0
```

Comment:
I think this line can be removed, since `stats['mean']` with get set to a value or zero in line 130.

Reply:
Agree !